lib/google_apis/generator/elixir_generator.ex

```diff
@@ -37,23 +37,43 @@ defmodule GoogleApis.Generator.ElixirGenerator do
   """
   @spec generate_client(ApiConfig.t()) :: {:ok, any()} | {:error, String.t()}
   def generate_client(api_config) do
-    Token.build(api_config)
-    |> load_models
-    |> set_model_filenames
-    |> update_model_properties
-    |> create_directories
-    |> write_model_files
-    |> load_global_optional_params
-    |> load_apis
-    |> set_api_filenames
-    |> write_api_files
-    |> write_connection
-    |> write_mix_exs
-    |> write_readme
-    |> write_license
-    |> write_gitignore
-    |> write_config_exs
-    |> write_test_helper_exs
+    token = Token.build(api_config)
+    if updated_discovery_revision?(token) do
+      token
```

Comment:
Rebind the variable here? Otherwise, we're not returning the possibly modified token (if that matters)

Reply:
I just changed it to return `:ok` since no one uses the return value anyway.